frontend/src/components/single-choice/single-choice.test.tsx

```diff
@@ -51,12 +84,43 @@ describe('SingleChoice', () => {
     fireEvent.click(buttonItem);
 
     await waitFor(() =>
-      expect(httpClient.post).toHaveBeenCalledWith('api/v1/answers', {
+      expect(httpClient.post).not.toHaveBeenCalledWith('api/v1/answers', {
```

Comment:
This is a negative test. For those you should use the minimum amount of detail. 
E.g. do not check the parameters for the method, check only the method, because if you make some mistake and call the method using nay other random parameter this spec won't catch it

Reply:
Thanks! I will remove the parameters, i need of `not` because the API no can be called when clicked in option